examples/flutter_gallery/lib/demo/material/slider_demo.dart

```diff
@@ -127,142 +199,243 @@ class _CustomValueIndicatorShape extends SliderComponentShape {
-        actions: <Widget>[MaterialDemoDocumentationButton(SliderDemo.routeName)],
-      ),
-      body: Padding(
-        padding: const EdgeInsets.symmetric(horizontal: 40.0),
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.spaceAround,
-          children: <Widget>[
-            Column(
-              mainAxisSize: MainAxisSize.min,
-              children: <Widget>[
-                Slider.adaptive(
-                  value: _value,
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 40.0),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.spaceAround,
+        children: <Widget>[
+          Column(
+            mainAxisSize: MainAxisSize.min,
+            children: <Widget>[
+              Semantics(
+                label: 'Editable numerical value',
+                child: Container(
+                  width: 48,
+                  height: 48,
+                  child: TextField(
+                    textAlign: TextAlign.center,
+                    onSubmitted: (String value) {
+                      final double newValue = double.tryParse(value);
+                      if (newValue != null && newValue != _continuousValue) {
+                        setState(() {
+                          _continuousValue = newValue.clamp(0, 100);
+                        });
+                      }
+                    },
+                    keyboardType: TextInputType.number,
+                    controller: TextEditingController(
+                      text: _continuousValue.toStringAsFixed(0),
+                    ),
+                  ),
+                ),
+              ),
+              Slider.adaptive(
+                value: _continuousValue,
+                min: 0.0,
+                max: 100.0,
+                onChanged: (double value) {
+                  setState(() {
+                    _continuousValue = value;
+                  });
+                },
+              ),
+              const Text('Continuous with Editable Numerical Value'),
+            ],
+          ),
+          Column(
+            mainAxisSize: MainAxisSize.min,
+            children: const <Widget>[
+              Slider.adaptive(value: 0.25, onChanged: null),
+              Text('Disabled'),
+            ],
+          ),
+          Column(
+            mainAxisSize: MainAxisSize.min,
+            children: <Widget>[
+              Slider.adaptive(
+                value: _discreteValue,
+                min: 0.0,
+                max: 200.0,
+                divisions: 5,
+                label: '${_discreteValue.round()}',
+                onChanged: (double value) {
+                  setState(() {
+                    _discreteValue = value;
+                  });
+                },
+              ),
+              const Text('Discrete'),
+            ],
+          ),
+          Column(
+            mainAxisSize: MainAxisSize.min,
+            children: <Widget>[
+              SliderTheme(
```

Comment:
The gallery has a custom dark-blue theme. Should the slider demo reflect that?

Would it be possible to configure this theme based on colors from the overall theme's color scheme?

Reply:
- Sorry - my mistake! This is the custom slider theme; it's supposed to be different.

Reply:
I think it correctly goes to a lighter blue for dark theme. Changed the custom track and tick colors to use colors from the color scheme